androidlibrary_lib/src/main/java/org/opendatakit/fragment/AlertDialogFragment.java

```diff
@@ -213,8 +223,8 @@ private void setTitle(String newTitle) {
    ///////////////////////////////////////////////////////////////////////////////////////
 
    public static AlertDialogFragment eitherReuseOrCreateNew(String alertDialogTag,
-       AlertDialogFragment inputAlertDialogFragment, FragmentManager fragmentManager,
-       boolean dismissActivity, int fragmentId, String title, String message) {
+                                                            AlertDialogFragment inputAlertDialogFragment, FragmentManager fragmentManager,
```

Comment:
Spacing seems to be weird ... why a huge indent?

Reply:
Resolved